homeassistant/components/aurora/__init__.py

```diff
@@ -21,9 +22,17 @@ async def async_setup_entry(hass: HomeAssistant, entry: AuroraConfigEntry) -> bo
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
+    entry.async_on_unload(entry.add_update_listener(update_listener))
     return True
 
 
+async def update_listener(hass: HomeAssistant, entry: AuroraConfigEntry) -> None:
+    """Handle options update."""
+    entry.runtime_data.threshold = int(
```

Comment:
This does update the threshold, but doesn't update the state of the sensors.

To apply the actual change, we should trigger a refresh or reload the config entry.

Reply:
Yes, that is a really good suggestion. I will add code for refreshing the state of the binary sensor.

Reply:
About the testing, I agree that there ought to be some tests added for the binary sensor. As it is now, I can only see that there are tests for the config flow.

But, I am sorry to say that I am not up to speed with writing tests yet so that fixing this would probably be to much of an effort for me. Sorry.

Reply:
So, I added the refresh, however now there are two commits. Not sure if that is ok, perhaps it should have been squashed into one before pushing?

Reply:
No need to squash, we squash and merge the pull requests.